proxy/src/main/java/com/velocitypowered/proxy/config/VelocityConfiguration.java

```diff
@@ -54,6 +54,14 @@
   @ConfigKey("online-mode")
   private boolean onlineMode = true;
 
+  @Comment({
+      "If client's ISP/AS sent from this proxy is different from the one from Mojang's",
+      "authentication server, the player is kicked. Effectively disallows players from",
+      "using proxies or VPNs when enabled."
+  })
+  @ConfigKey("prevent-client-proxy-connections")
+  private boolean preventClientProxyConnections = false;
```

Comment:
Previous code defaulted to passing the IP param, should this be `true` instead?

Reply:
Vanilla has this off by default, so I went with false.

Reply:
I'm inclined to say we should leave this off, as `prevent-proxy-connections` is a rather weak form of protection that can be easily bypassed.